.travis.yml

```diff
@@ -1,4 +1,5 @@
 sudo: false
+dist: trusty
```

Comment:
because hhvm?
 https://docs.travis-ci.com/user/languages/php/#choosing-php-versions-to-test-against

clearly, I'm confused. what's the right config and why?